query-engine/connector-test-kit/src/test/scala/writes/ids/relation_pks/CompoundPKRelationFieldSpec.scala

```diff
@@ -1175,3 +1175,138 @@ class CompoundPKRelationFieldSpec extends FlatSpec with Matchers with ApiSpecBas
     res11.toString() should be("{\"data\":{\"updateChild\":{\"id\":3,\"parents\":[]}}}")
   }
 }
+
```

Comment:
No more scala tests, all new test must be created in the rust test suite.

Reply:
Removed the test and added a simpler Rust one. Also confirmed all existing tests are running with the preview flag already. So only additional testing for the new compound behaviour was added.